video/out/opengl/ra.h

```diff
@@ -5,6 +5,7 @@
 
 // Handle for a rendering API backend.
 struct ra {
+    const char *name;       // unique name for this RA implementation
```

Comment:
Is you insist on this, could you either make it 1. correctly dependent on the Vulkan driver, or 2. a field of ra_fns. (Even if it isn't a function.)

Reply:
Come to think of it, once we switch to glslang, we'll need a custom header on cached_program anyway - since we also need to cache the generated SPIR-V, not just the VkPipeline.

Reply:
> Should we add magic numbers to the ra_gl/ra_vk headers, though?

I guess ra_gl could easily add a magic number.